src/ray/protobuf/experimental/instance_manager.proto

```diff
@@ -42,78 +28,6 @@ message GetAvailableInstanceTypesResponse {
   int64 available_instances = 2;
 }
 
-// Represents the state of a launched instance.
-// An instance is considered launched as long as
-// it has a unique instance_id associated with it.
-//
-// Note a launched instance may be DEAD. In this case,
-// the state will be garbage collected after some timeout
-// period (by default 30 minutes).
-message Instance {
```

Comment:
From product side, I don't have concern about reusing it here, but it might be hard for you in the future where if your instance manager implementation detail changes, you would need to change the `autoscaler.proto`. 

Suggesting to have an instance struct for `instance_manager.proto`, and a common denominator instance_gcs struct for `autoscaler.proto`, which only contains the observability required from gcs. It is totally up to you though.